packages/fxa-auth-server/fxa-oauth-server/test/api.js

```diff
@@ -917,6 +919,28 @@ describe('/v1', function() {
             });
         });
 
+        it('returns an implicit JWT formatted token', async function() {
+          mockAssertion().reply(200, VERIFY_GOOD);
+          const res = await Server.api.post({
+            url: '/authorization',
+            payload: authParams({
+              client_id: jwtClient.id,
+              response_type: 'token',
+            }),
+          });
+
+          const defaultExpiresIn = config.get('expiration.accessToken') / 1000;
+          assert.equal(res.statusCode, 200);
+          assertSecurityHeaders(res);
+          assert(res.result.access_token);
+          assert.include(res.result.access_token, '.');
```

Comment:
As above, naive readers would probably appreciate a comment that this dot uniquely indicates a JWT.